README.md

```diff
@@ -242,53 +240,53 @@ different lengths. *Higher is better*
 
 | String length             | xxhashjs 32-bit    | xxhashjs 64-bit    | xxhash-wasm 32-bit      | xxhash-wasm 64-bit      |
 | ------------------------: | ------------------ | ------------------ | ----------------------- | ----------------------- |
-| 1 byte                    | 513,517 ops/sec    | 11,896 ops/sec     | 2,122,322 ops/sec       | ***3,556,437 ops/sec*** |
-| 10 bytes                  | 552,133 ops/sec    | 12,953 ops/sec     | 2,075,154 ops/sec       | ***3,196,204 ops/sec*** |
-| 100 bytes                 | 425,277 ops/sec    | 10,838 ops/sec     | 1,867,002 ops/sec       | ***3,169,921 ops/sec*** |
-| 1,000 bytes               | 102,165 ops/sec    | 6,697 ops/sec      | 1,468,336 ops/sec       | ***2,393,906 ops/sec*** |
-| 10,000 bytes              | 13,010 ops/sec     | 1,452 ops/sec      | 446,415 ops/sec         | ***787,430 ops/sec***   |
-| 100,000 bytes             | 477 ops/sec        | 146 ops/sec        | 58,339 ops/sec          | ***91,895 ops/sec***    |
-| 1,000,000 bytes           | 36.40 ops/sec      | 12.93 ops/sec      | 5,443 ops/sec           | ***8,379 ops/sec***     |
-| 10,000,000 bytes          | 3.12 ops/sec       | 1.19 ops/sec       | 307 ops/sec             | ***417 ops/sec***       |
-| 100,000,000 bytes         | 0.31 ops/sec       | 0.13 ops/sec       | 28.70 ops/sec           | ***35.63 ops/sec***     |
-
-`xxhash-wasm` outperforms `xxhashjs` significantly, the 32-bit is up to 98 times
+| 1 byte                    | 513,517 ops/sec    | 11,896 ops/sec     | 5,752,446 ops/sec       | ***4,438,501 ops/sec*** |
+| 10 bytes                  | 552,133 ops/sec    | 12,953 ops/sec     | 6,240,640 ops/sec       | ***4,855,340 ops/sec*** |
+| 100 bytes                 | 425,277 ops/sec    | 10,838 ops/sec     | 5,470,011 ops/sec       | ***4,314,904 ops/sec*** |
```

Comment:
The 32-bit should be bold, not the 64-bit for up to 100 bytes, as it was mentioned in the text below, although the cross-over point for the 64-bit to become faster is almost exactly at 1,000 bytes or very slightly below it (not 100).

Reply:
Ah, right. Fixed. My experience was that there was actually a pretty high degree of variability in those small-byte-size benchmarks—I wouldn't be particularly confident in that 100 vs. 1000 measurement, but might as well summarize the data in the table.